.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+## What did you implement?
+
+_A summary of the change from the implementation perspective that could help
+reviewers understanding the whole solution_
+
+## Checklist:
+
+- [ ] I exported my component in [src/index.ts](https://github.com/VirtusLab/tetrisly-react/blob/main/src/index.ts) file.
```

Comment:
I would add the following checks: 
```suggestion
- [ ] I exported my component in [src/index.ts](https://github.com/VirtusLab/tetrisly-react/blob/main/src/index.ts) file.
- [ ]  I described colors, spacing, typography, and similar attributes with Figma Tokens and they are placed in the config object in the `*.styles` file
- [ ] I tested the component (including custom prop)
- [ ] I've written a storybook story for the component
```

@adrian-potepa would you add anything?

Reply:
Yep, it will be awesome 👌

Reply:
Ok, I added suggested checkboxes ;-)